scripts/create-dumps.sh

```diff
@@ -1,41 +1,46 @@
 #!/usr/bin/env bash
 
-# NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE 
-# This is a saved script that is running in production. We don't know where
-# this might be kept in repo, so we'e checking a version in here for safe keeping.
-# This script will NOT work outside the gcloud hosted bookbrainz.org site. 
-# It will need to be adapted for the Hetzner production setup.
-# NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE NOTE 
+source /home/bookbrainz/bookbrainz-site/scripts/config.sh
 
 # Switch directory
-pushd /home/bookbrainz/bookbrainz.org/bookbrainz-docker-prod/dumps
+pushd /home/bookbrainz/data/dumps
+
 
 DUMP_FILE=bookbrainz-dump-`date -I`.sql
 
-export PGPASSWORD="postgres"
+echo "Creating data dump..."
+export PGPASSWORD="bookbrainz"
```

Comment:
I was hoping we'd get rid of a hardcoded password entirely…
Should this be set as an env variable instead?

Reply:
Our other databases do not have passwords -- we should probably work to get rid of this one too.